server.js

```diff
@@ -1,5 +1,27 @@
-const app = require('./app')
-
+import app from "./app.js";
 app.listen(3000, () => {
-  console.log("Server running. Use our API on port: 3000")
-})
+  console.log("Server running. Use our API on port: 3000");
+});
+import mongoose from "mongoose";
+
+// Connection string do twojej bazy danych MongoDB w MongoDB Atlas
+const dbConnectionURI =
+  "mongodb+srv://MonikaG:Mm29030405@cluster0.eeog9dx.mongodb.net/db-contacts";
```

Comment:
Use .env file and dotenv aby móc używać zmiennych środowiskowych. Następnie użyj .env.example aby poinformować jakie klucze są potrzebne. np `DB_URL=` Ten drugi plik wyślij do repo a ten pierwszy ma być tylko lokalnie zrobiony.